actions/get_fabric_ips.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: get_fabric_ips
+pack: hpecfm
+description: Get an array of switchesfabric ip addresses from the plexxi controller
+runner_type: python-script
+entry_point: get_fabric_ips.py
+enabled: true
+parameters:
+  ipaddress:
+    type: "string"
+    description: "The IP address of a plexxi controller"
```

Comment:
What's "plexxi"? "cfm" after spelling it out?

Reply:
replacing plexxi with hpecfm